### PR TITLE
Feature/playwright account

### LINKED ISSUE #2840: Migrate integration/account tests to playwright

### PR DESCRIPTION
**Pull request by clari182**
- closes #2840 
- part of #2806

### CODE REVIEW

**Review by cesarvarela — CHANGES_REQUESTED** [review]
Please merge staging, playwright workflow should be green now.

**Comment by pdcp1** [comment]
Re-running checks since these 3 tests were skipped in the last run. These 3 tests should be executed now:

image

**Comment by pdcp1** [comment]
> Re-running checks since these 3 tests were skipped in the last run. These 3 tests should be executed now:
> 
> image

They are still being skipped. @clari182 please merge staging and check if all Account tests pass

**Review by pdcp1 — CHANGES_REQUESTED** [review]
Some tests are failing
image

File: site/gatsby-site/session.json
Comment (pdcp1) [inline site/gatsby-site/session.json:1]: Is this file needed for the account test file?

**Review by pdcp1 — CHANGES_REQUESTED** [review]

**Review by pdcp1 — APPROVED** [review]

**Pull request merged**